packages/client/components/NewMeetingTeamPicker.tsx

```diff
@@ -47,7 +50,7 @@ const NewMeetingTeamPicker = (props: Props) => {
   }
 
   const handleAddTeamClick = () => {
-    window.open(`/newteam/1`, '_blank', 'noreferrer')
+    setAddTeamDialogOpen(true)
```

Comment:
-1 Since we're no longer pushing a page, we're going to lose the page view event for the new team page.  We should add a "Add Team Clicked" event here so we don't lose coverage.